orca-redis/orca-redis.gradle

```diff
@@ -2,6 +2,7 @@ apply from: "$rootDir/gradle/kotlin.gradle"
 apply from: "$rootDir/gradle/spock.gradle"
 
 dependencies {
+  compile project(":orca-core")
```

Comment:
What's the advantage of calling this out explicitly if it's a dependency of front50? Just explicitness?

Reply:
yeah pretty much - directly needs stuff in orca core (and nothing in orca-fiddy)

Reply:
explicitness is goodness